build/build-in-docker

```diff
@@ -72,12 +68,12 @@ $DOCKER_CMD run $DOCKER_GPU_OPTS -it -u $(id -u):$(id -g) --rm \
   -v "/etc/passwd:/etc/passwd:ro" \
   -v "/etc/shadow:/etc/shadow:ro" \
   -v "/etc/sudoers.d:/etc/sudoers.d:ro" \
-  -v "$REPODIR:$WORKSPACE_REPODIR:rw" \
-  -v "$LOCAL_CCACHE_DIR:$WORKSPACE_CCACHE_REPODIR:rw" \
-  -v "$LOCAL_MAVEN_REPO:$WORKSPACE_MAVEN_REPODIR:rw" \
-  --workdir "$WORKSPACE_REPODIR" \
+  -v "$REPODIR:$REPODIR:rw" \
+  -v "$LOCAL_CCACHE_DIR:$LOCAL_CCACHE_DIR:rw" \
+  -v "$LOCAL_MAVEN_REPO:$LOCAL_MAVEN_REPO:rw" \
+  --workdir "$REPODIR" \
```

Comment:
@jlowe The reason I asked about ccache is that now we have the same input paths between host and docker container